endpoints/s3/src/abortmultipartupload.cpp

```diff
@@ -162,22 +165,33 @@ void irods::s3::actions::handle_abortmultipartupload(
 		for (std::filesystem::directory_iterator iter{part_file_location}; iter != end; ++iter) {
 			if (std::filesystem::is_regular_file(*iter)) {
 				if (std::regex_match(iter->path().filename().string(), part_files_regex)) {
-					std::filesystem::remove(iter->path());
+					try {
+						std::filesystem::remove(iter->path());
+					}
+					catch (std::exception&) {
+						logging::error(
+							"{}: Failed to remove part_file [{}].", __func__, iter->path().filename().string());
+						response.result(beast::http::status::internal_server_error);
+					}
 				}
 			}
 		}
 	}
 	catch (std::exception&) {
+		logging::error(
+			"{}: Upload ID [{}] - Exception caught when iterating through part files for removal.",
+			__func__,
+			upload_id);
```

Comment:
May want to include the `what()` in this message

Reply:
done